spec/models/favorite_spec.rb

```diff
@@ -10,6 +10,16 @@
     end
   end
 
+  describe "#check_pet" do
```

Comment:
Seems like there were a few more model methods that didn't get tested here, let's just make sure they don't get forgotten.

Reply:
could you list those out here explicitly so I can make a new card in projects?